jenkins-2.yaml

```diff
@@ -14,6 +16,8 @@ package:
       - glibc-locale-en
       # The entrypoint script is pulled from a separate repo, which we'e packaged.
       - jenkins-entrypoint
+      - openjdk-17
```

Comment:
This is intentional. Today we define these as runtime deps at the image level - we talked about moving these to package level definitions recently.